test/python/test_dagcircuit.py

```diff
@@ -498,7 +498,7 @@ def test_circuit_depth(self):
 
     def test_circuit_width(self):
         """Test number of qubits in circuit."""
-        self.assertEqual(self.dag.width(), 6)
+        self.assertEqual(self.dag.num_qubits(), 6)
```

Comment:
Can you update the test name, and add a test for `dag.width`? (Feel free to add a `ClassicalRegister` to `circ` in `setup`, so that the value of `width` will be different than that of `num_qubits`.)

Reply:
Now looking at this change

Reply:
Ha, in test/python/test_dagcircuit.py  `TestCircuitProperties` sets up with no classical bits, so both tests are the same for now!

Reply:
Change pushed.